### PR TITLE
Catch2 Homework

### PR DESCRIPTION
**Pull request by jakjan95**
Authors: @jakjan95 @MaciejSalamonski @McKucia @Pawellos

### CODE REVIEW

File: bowling/bowling.ut.cpp [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+    }
+}
+
+SCENARIO("Getting 1 points in every game", "[points]") {
+    GIVEN("Ten turns of game") {
+        std::vector<int> game;
+        for (size_t i = 0; i < 10; i++) {
+            game.emplace_back(1);
+            game.emplace_back(0);
+        }
```
Comment (WBigaj) [inline bowling/bowling.ut.cpp:32]: You can try using std::generate here.

File: bowling/bowling.ut.cpp [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+    }
+}
+
+SCENARIO("First ball - 9, Second - 0", "[points]") {
+    GIVEN("Ten turns of game") {
+        std::vector<int> game(10, 9);
+
+        WHEN("9 pins hit with first ball and 0 with second one") {
+            auto gameScore = score(90);
```
Comment (WBigaj) [inline bowling/bowling.ut.cpp:85]: Firsly you are capturing int from score function. In the rest of test you capturing result via auto. Why?

**Review by WBigaj — COMMENTED** [review]
I have checked your code and left my feedback.

**Review by ziobron — APPROVED** [review]
2 test cases are duplicated IMO (Game with one strike in middle, Game with one spare in middle). They are covered by other cases

**Comment by ziobron** [comment]
{ "requirement_ids": [4598], "users": ["jakjan95", "MaciejSalamonski", "McKucia", "Pawellos"] }

**Pull request closed without merging**